lib/kaguya/core.ex

```diff
@@ -68,7 +68,7 @@ defmodule Kaguya.Core do
     {:noreply, %{socket: socket}}
   end
 
-  defp reconnect(_tries \\ 0) do
```

Comment:
At least for now I don't want to remove this, since a user might want to control how many times the bot attempts to reconnect before giving up.